docs/_data/components/schemas/temporary-aar-exception.yaml

```diff
@@ -0,0 +1,32 @@
+type: object
+additionalProperties: false
+description: |
+  Data structure for a temporary AAR Exception against a resource.
+  Includes a unique ID, start and (optional) end time, and a reason.  
```

Comment:
Is `reason` meant to be required?

Reply:
No.  Of course it is required, but the question is _when_.  Is it truly required when the exception is created, or at the point at which the reason can no longer be modified, such as 5 days in the past (I believe that's the MISO rule).  My thinking here is that any validation of that is a little too much for JSON schema and has to be left to the server.